src/locales/sv/translation.json

```
@@ -673,5 +673,55 @@
     "whyText3": "Vi följer Dataskyddsförordningen och de andra lagar och regler som gäller på dataskyddsområdet. Du har alltid rätt att begära korrigering eller radering av dina personliga uppgifter.",
     "contactTitle": "Kontakta Oss",
     "contactText1": "Om du har frågor eller önskemål kring hantering av dina personuppgifter, vänligen kontakta oss."
+  },
+  "learnMorePage": {
+    "title": "Globala utsläpp",
+    "subtitle": "Varför mätning, rapportering och transparens av utsläpp är kritisk för att uppnå verklig förändring.",
+    "impactTitle": "Förstå påverkan",
+    "impactDescription": "Varje partikel representerar miljontals ton CO₂-utsläpp. Titta hur de ackumuleras över tid, vilket visar den exponentiella tillväxten av globala utsläpp sedan industriella revolutionen.",
+    "impactDescription2": "Denna visualisering visar hur utsläppen har ökat dramatiskt sedan 1950, med en särskilt snabb tillväxt i senare decennier.",
+    "trackingTitle": "Mätning för att hantera",
+    "trackingDescription": "Vi kan inte effektivt minska vad som inte mäts. Noggrann utsläppsmätning är grunden för meningsfull klimatåtgärd.",
+    "companies": "Företag",
+    "companiesDescription": "Företagens utsläpp utgör en betydande del av världens koldioxidutsläpp. Noggrann mätning gör att företag kan identifiera minskningsmöjligheter.",
+    "communities": "Samhällen",
+    "communitiesDescription": "Lokala samhällen behöver förstå sin koldioxidmängd för att kunna implementera effektiva minskningsstrategier och hållbara praktiker.",
+    "countries": "Länder",
+    "countriesDescription": "Nationella utsläppsdata är kritiska för att ställa och uppnå klimatmål, och att hålla länder ansvariga för sina åtaganden.",
+    "dataGaps": "Dataluckor",
+    "dataGapsDescription": "Saknade eller felaktiga utsläppsdata hindrar våra förmågor att fatta informerade beslut och ta meningsfull klimatåtgärd.",
+    "whyTrackingMatters": "Varför mätning är viktigt",
+    "whyTrackingMattersDescription": "Komplett utsläppsmätning är essentiell för:",
+    "trackingBenefits": [
+      "Att sätta vetenskapligt baserade minskningsmål",
+      "Att identifiera de mest effektiva minskningsstrategierna",
+      "Att övervaka framsteg över tid",
+      "Att säkerställa ansvarsskyldighet och transparens",
+      "Att underlätta informerade policy- och regleringsbeslut"
+    ],
+    "csrdTitle": "Framtiden för utsläppsrapportering",
+    "csrdDescription": "EU:s direktiv om företagens hållbarhetsrapportering (CSRD) representerar ett betydande steg framåt mot standardiserad och omfattande utsläppsrapportering.",
+    "currentChallenges": "Historiska rapporteringsutmaningar",
+    "currentChallengesDescription1": "Inkonsekventa metoder mellan företag och regioner",
+    "currentChallengesDescription2": "Begränsad omfattning av utsläppstäckning",
```

```suggestion
    "currentChallengesDescription2": "Begränsad täckning av utsläpp",
```

A bit unsure tho, doesn't feel like to correct way to put it but at least an improvement imo